pkg/storage/ingest/reader.go

```diff
@@ -829,8 +829,12 @@ type concurrentFetchers struct {
 	lastReturnedRecord int64
 }
 
+// defaultMinBytesWaitTime is the time the Kafka broker can wait for MinBytes to be filled.
+// This is usually used when there aren't enough records available to fulfil MinBytes, so the broker waits for more records to be produced.
+var defaultMinBytesWaitTime = 10 * time.Second
```

Comment:
I think we need to move this next to the top of the fiile.

Reply:
in #9202 it's now a constant and at the top